spdmlib/src/common/session.rs

```diff
@@ -851,7 +852,17 @@ impl SpdmSession {
                 }
             }
             _ => panic!("unknown session state"),
+        };
+
+        if let Err(err) = r {
+            if err.status_code
+                == crate::error::StatusCode::CRYPTO(StatusCodeCrypto::SEQUENCE_NUMBER_OVERFLOW)
+            {
+                self.set_default();
```

Comment:
Can set_default() free all resources? such as session_id.

Reply:
> Can set_default() free all resources? such as session_id.

Yes, it can, set_default will free all resources, session_id will be set to invalid one, value 0.